aea/cli/utils/package_utils.py

```diff
@@ -440,14 +441,46 @@ def is_item_present(
     :return: boolean is item present.
     """
     # check item presence only by author/package_name pair, without version.
-    item_type_plural = item_type + "s"
-    items_in_config = set(
-        map(lambda x: (x.author, x.name), getattr(ctx.agent_config, item_type_plural))
-    )
+
     item_path = get_package_path(ctx, item_type, item_public_id, is_vendor=is_vendor)
-    return (item_public_id.author, item_public_id.name,) in items_in_config and Path(
-        item_path
-    ).exists()
+    is_item_registered = bool(
+        get_item_public_id_by_author_name(
+            ctx.agent_config, item_type, item_public_id.author, item_public_id.name
+        )
+    )
```

Comment:
let's make it explicit? `is_item_registered = result is not None`

Reply:
done